playground/AWS/AWS.AppHost/appsettings.json

```diff
@@ -3,7 +3,8 @@
     "LogLevel": {
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning",
-      "Aspire.Hosting.Dcp": "Warning"
+      "Aspire.Hosting.Dcp": "Warning",
+      "Microsoft.Extensions.Diagnostics.HealthChecks.DefaultHealthCheckService": "None"
```

Comment:
Add this here instead:

https://github.com/dotnet/aspire/blob/b5091e330838bdf402d52ebfbb6a31b546f36b4c/src/Aspire.Hosting/DistributedApplicationBuilder.cs#L129-L136